lib/types/elasticsearch.d.ts

```diff
@@ -0,0 +1,60 @@
+export interface Plugin {
```

Comment:
I think declaration files are not only for ambient declarations but also for indicating itself only contains types. Please let me know if you have specific documentation.

Reply:
I see, it’s good to indicate only type contained in file. Okay.
(I don’t know any documents about extension of typescript, I thought it because I never see examples uses ‘.d.ts’ to define new types/interfaces.